src/class_singleCleaner.py

```diff
@@ -36,11 +36,12 @@
 
 
 class singleCleaner(StoppableThread):
+    """Module run singleCleaner thread"""
     name = "singleCleaner"
     cycleLength = 300
     expireDiscoveredPeers = 300
 
-    def run(self):
+    def run(self):        # pylint: disable=too-many-branches, protected-access
```

Comment:
two spaces is enough

Reply:
Okay, I will remember from next time, and I will change as requested.